src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -527,7 +528,7 @@ int goto_instrument_parse_optionst::doit()
 
     if(cmdline.isset("list-symbols"))
     {
-      show_symbol_table(true);
+      show_symbol_table(symbol_table, get_ui());
```

Comment:
`--list-symbols` seems to be an alias of `--show-symbol-table`. I suggest to factorise the calls and deprecate one of the options if not needed anymore.

Reply:
The behaviour of "list-symbols" and "show-symbol-table" ought to be different - I believe the above change thus introduces a bug.

Reply:
Having read the rest of the PR: this should be `show_symbol_table_brief` instead.

Reply:
indeed